### ISSUE TITLE
Pyanalyze outputs apparently harmless warning message when run on any file

### ISSUE THREAD

**Issue by CAM-Gerlach** [quora/pyanalyze#221]
I get the following message printed to my terminal whenever running pyanalyze on any non-trivial files in my project, once, for the first file Pyanalyze sees. As no traceback or other information is printed, I'm not sure where its coming from. However, it is not considered an error, as pyanalyze still exits with code 0, and only ever prints once per path spec, but its a minor annoyance and I figured it would be worth information you of. Thanks!

```
Cannot overload a class or an imported name: NameInfo(name='CodeType', is_exported=False, ast=ImportedName(module_name=('types',), name='CodeType'), child_nodes=None)
```

**Comment by JelleZijlstra** [comment]
It's from https://github.com/JelleZijlstra/typeshed_client and I should really get around to figure out what's causing it. Thanks for the reminder.

**Comment by CAM-Gerlach** [comment]
Sure thing!

**Comment by JelleZijlstra** [comment]
I just published typeshed-client 1.1.1 which fixes this. I am not bumping pyanalyze's version requirement because this error is harmless, but upgrading the typeshed-client dependency should make the warning go away.

**Comment by CAM-Gerlach** [comment]
Thanks, tested and works great! Any plans for a `pyanalyze` release soon, so I don't have to depend on Git checkouts in pre-commit, requirements, install_requires and CI?

**Comment by JelleZijlstra** [comment]
Just released 0.3.0.

**Comment by CAM-Gerlach** [comment]
Awesome @JelleZijlstra ! Thanks!
